File: examples/pub.py
import sys
import requests


def main():

    chan, msg = sys.argv[1], sys.argv[2]
    url = "https://streamboat.tv/pub/{}".format(chan)
    r = requests.post(url, data=msg, headers={'Authorization': "Token 12345"})
    r.raise_for_status()
    print r.json()


if __name__ == '__main__':
    main()

File: examples/pub.js
var request = require("request"),
     chan = process.argv[2],
     msg = process.argv[3];

request({
    url: "https://streamboat.tv/pub/" + chan,
    method: "POST",
    headers: {"Authorization": "Token 12345"},
    body: msg,
    },
    function(err, res, body) {
        console.log(body);
    }
);